projects/aca-content/src/lib/store/initial-state.ts

```diff
@@ -56,7 +56,8 @@ export const INITIAL_APP_STATE: AppState = {
     status: {
       isQuickShareEnabled: true
     }
-  } as any
+  } as any,
+  searchItemsTotalCount: null
```

Comment:
Can you check if these changes still work on the latest develop, or need to be moved to a different file? I think this file was completely wiped recently, with just a single export statement as part of #3952 ([changes](https://github.com/Alfresco/alfresco-content-app/pull/3952/files#diff-c27969366fa9b017d410c110b00cf751e04db55158fcc58b4c95edafce5a0664L27) here)

Reply:
thank you for this catch!
I will try to update all our branches with the latest changes.
It's not easy to develop a feature without merging into development and keep it only in feature branches when there are so many important architecture changes in 3 repositories.
There are 3 developers who are working on the feature and we have a lot of nested branches.